dev-util/muon/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person" proxied="yes">
+    <email>arsen@aarsen.me</email>
+    <name>Arsen Arsenović</name>
+  </maintainer>
+  <maintainer type="project" proxied="proxy">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <use>
+    <flag name="libpkgconf">
```

Comment:
I'll sort these two, but wonder if we should rename libpkgconf->pkg-config given it more represents the _feature_.

I'll leave it as-is for now. Thanks a lot!

Reply:
I called it this since it can't be provided by either pkg-config implementation, and didn't want to make the flag confusing. Either works, though